test/apps/iodemo/io_demo.cc

```diff
@@ -2435,7 +2477,13 @@ class DemoClient : public P2pDemoCommon {
                     const char *tail = (op == IO_OP_MAX) ? "" : " | ";
 
                     conn_log << name << " " << mbs << "MBs " << "iops: "
-                             << iops << tail;
+                             << iops;
+                    if (opts().window_size == 1) {
+                        conn_log << " max latency: " << max_lat * 1e6 <<"usec "
+                                 << " avr latency: "
+                                 << (tot_lat / num_completed) *1e6 << "usec";
+                    }
+                    conn_log << tail;
```

Comment:
1. pls run this through clang-format style check
2. "usec" -> "us" to make line shorter
3. print "max-lat: " and "avg-lat:" to make line shorter
4. space before "us"
5. there's one extra space before "avr" (that should be "avg")

Reply:
done